lib/util/macros/src/lib.rs

```diff
@@ -9,7 +9,7 @@ use syn::token::Return;
 use syn::{braced, bracketed, Expr, Ident, Result, Token};
 
 // 1 min
-const DEFAULT_TIMEOUT: u64 = 60 * 1000;
+const DEFAULT_TIMEOUT: u64 = 45 * 1000;
```

Comment:
It's a bit hard to know what this magic number means when it's beside the doc saying "1 min", is there a way to make this a bit more clear? Maybe even having a link to the new API_TIMEOUTS.md doc.